kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/Feature.java

```diff
@@ -11,6 +11,7 @@
       .filter(Predicate.not(List::isEmpty))
       .isPresent()
   ),
+  KSQL_DB(cluster -> cluster.getKsqldbServer() != null),
```

Comment:
You should also add it here: 
https://github.com/provectus/kafka-ui/blob/master/kafka-ui-contract/src/main/resources/swagger/kafka-ui-api.yaml#L1371

Reply:
already done